apps/game_backend/priv/repo/migrations/20240812135550_add_active_to_map_config.exs

```diff
@@ -0,0 +1,9 @@
+defmodule GameBackend.Repo.Migrations.AddActiveToMapConfig do
+  use Ecto.Migration
+
+  def change do
+    alter table(:map_configurations) do
+      add :active, :boolean, default: :false
+    end
```

Comment:
Merging this migration as it is and with the filtering for `active == true` will cause the game to break until there is at least one map configuration with `active == true`